types/jest/index.d.ts

```diff
@@ -757,8 +758,25 @@ declare namespace jest {
         toMatch(expected: string | RegExp): R;
         /**
          * Used to check that a JavaScript object matches a subset of the properties of an object
+         *
+         * Optionally, you can provide an object to use as Generic type for the expected value.
+         * This ensures that the matching object matches the structure of the provided object-like type.
+         *
+         * @example
+         *
+         * type House = {
+         *   bath: boolean;
+         *   bedrooms: number;
+         *   kitchen: {
+         *     amenities: string[];
+         *     area: number;
+         *     wallColor: string;
+         *   }
+         * };
+         *
+         * expect(desiredHouse).toMatchObject<House>(...standardHouse, kitchen: {area: 20}) // wherein standardHouse is some base object of type House
```

Comment:
Is the intent to spread into an object here?
```suggestion
         * expect(desiredHouse).toMatchObject<House>({ ...standardHouse, kitchen: {area: 20} }) // wherein standardHouse is some base object of type House
```

Reply:
This was indeed meant to be an object spread. I indeed forgot a `{`. Looks like the PR was merged already though so this'll have to be covered in a next one. That said... It's not *really* important I think so I think I'll just wait for the next PR to come up and the bot will ping me now as well so I can request the change then from whoever is editing.